README.md

```diff
@@ -18,12 +18,24 @@ for the frontend extension.
 
 ## Install
 
-Note: You will need NodeJS to install the extension.
+You will need NodeJS to install the extension. For JupyterLab 4.x you will need NodeJS>=v20 and >=v14 for JupyterLab 3.x.
+
+### JupyterLab 4.x
 
 ```bash
 pip install rucio-jupyterlab
 ```
 
+### JupyterLab 3.x
+
+For JupyterLab 3.x, please use the last supported version.
+
+```bash
+pip install rucio-jupyterlab==0.9.10
+```
+
+#### Download mode
+
```

Comment:
@ftorradeflot is the section on ipykernel still relevant for the new version ? (line 51 of this file - sorry)

Reply:
yes, this is still relevant